packages/liferay-js-insights/reporters/table.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const Table = require('cli-table');
+
+/**
+ * Prints the modules's insights report as a table in the terminal. By default, it only includes the `app`, `module`, `clay3` and `react` fields for brevity.
+ */
+module.exports = async function(modulesInfo) {
+	const table = new Table({
+		colWidths: [35, 35, 50, 70],
```

Comment:
Are these suggested widths or hard min/max limits? Just asking because we could probably calculate optimal widths if we wanted to avoid wasted space.

Reply:
Just manual trial and error on my screen. And we all know which screen matters here ;)